manala.sensu_go/handlers/main.yml

```diff
@@ -0,0 +1,13 @@
+---
+
+- name: sensu go restart
+  service:
+    name:  "{{ item }}"
+    state: restarted
+  with_items: "{{
```

Comment:
You should have two distincts handler (`sensu go agent restart` and `sensu go backend restart`) and call the right handler in specific tasks.